src/features/cell-renderers/dateCellRenderer.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import * as React from 'react';
+import baseCellRenderer from './baseCellRenderer';
+import type { CellRendererParams } from './flowTypes';
+import ReadableTime from '../../components/time/ReadableTime';
+
+const dateCellRenderer = (cellRendererParams: CellRendererParams) =>
+    baseCellRenderer(cellRendererParams, cellValue => (
+        <ReadableTime timestamp={Date.parse(cellValue)} alwaysShowTime />
```

Comment:
Can we name this readableTimeCellRenderer? It doesn't fit the use case for shield since we need the year of it too and ReadableTime seems to show relative.

Reply:
ReadableTime can show the year.

Reply:
> Can we name this readableTimeCellRenderer? It doesn't fit the use case for shield since we need the year of it too and ReadableTime seems to show relative.

ReadableTime can show the year. It shows relative time only when the time is less in an hour by default.